### PR TITLE
Review CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ksenteer**
Review CONTRIBUTING.md for:

* consistency
* language quality
* compliance with HERE documentation standards and best practices of technical writing

Relates-to: OLPEDGE-622

Signed-off-by: Kateryna Osadchenko <ext-kateryna.osadchenko@here.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,19 +35,19 @@ enum class FetchOption {
 
 ### File Headers
 
-Every `.cpp`, `.h` and `.inl` file must contain the following header at the very top:
+Make sure every `.cpp`, `.h` and `.inl` file contains the following header at the very top:
 
 ```cpp
 /*
```
Comment (andescu) [inline CONTRIBUTING.md]: This licence headers comes from our OSS office and was provided by the legal department, means we cannot modify it. Please change back to original version (even if it contains typos :( )

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -56,9 +56,9 @@ Every `.cpp`, `.h` and `.inl` file must contain the following header at the very
  * SPDX-License-Identifier: Apache-2.0
  * License-Filename: LICENSE
  */
-```
+`"
```
Comment (andescu) [inline CONTRIBUTING.md]: ' ' ' is needed to close ' ' 'cpp

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -56,9 +56,9 @@ Every `.cpp`, `.h` and `.inl` file must contain the following header at the very
...
 
-CMake files or bash scripts must contain the following header at the very top:
+Make sure CMake files or bash scripts contain the following header at the very top:
 
 ```cmake
 # Copyright (C) 2019 HERE Europe B.V.
```
Comment (andescu) [inline CONTRIBUTING.md]: same here

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -67,36 +67,36 @@ CMake files or bash scripts must contain the following header at the very top:
...
+# Unless required by the applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
-```
+`"
```
Comment (andescu) [inline CONTRIBUTING.md]: same

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -105,26 +105,26 @@ find foo/bar/ -iname *.h -o -iname *.cpp | xargs clang-format -i --style=file
 ### Design Guidelines
 
 * Follow the **SOLID** design principles:
-  <br />**Single responsibility** - a class should not have more than one reason to change.
-  <br />**Open/closed principle** - software should be open for extension by identifying customization points and closed for modification by making small parts well-defined and composable.
-  <br />**Liskov substitution principle** - subtypes of a class or interface should not break the API contract of that class or interface.
+  <br />**Single responsibility** - a class cannot have more than one reason for a change.
+  <br />**Open/closed principle** - keek software open for extension by identifying customization points and closed for modification by making small parts well-defined and composable.
```
Comment (andescu) [inline CONTRIBUTING.md]:
```suggestion
  <br />**Open/closed principle** - keep software open for extension by identifying customization points and closed for modification by making small parts well-defined and composable.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -152,35 +152,35 @@ The SDK uses the following pattern for namespace name: **olp**::*component::subc
 
 #### API Class Name
 
-The SDK uses the following pattern for API class name : *Api***Client**, e.g. `StreamLayerClient`
+The SDK uses the following pattern for API class name: *Api***Client**, for example, `StreamLayerClient`.
 
 #### API Method Signature
 
-The SDK provides a set of common classes that are used when defining the APIs.
+The SDK provides a set of standard classes that are used to define the APIs.
```
Comment (andescu) [inline CONTRIBUTING.md]: `common` is a more used term for such classes, or `generic`
`standard` might be mistaken with the std library classes which is something totally different.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -152,35 +152,35 @@ The SDK uses the following pattern for namespace name: **olp**::*component::subc
...
 |Base class|Description|
 |------|------|
-|**ApiResponse**|Template class representing the outcome of making a request. It either contains the result or the error.|
+|**ApiResponse**|Template class represents the outcome of when you make a request. It either contains the result or the error.|
 |**ApiError**|Error information container.|
 |**ErrorCode**|Common error codes holder.|
-|**CancellationToken**|Control object for cancelling asynchronous requests.|
-|**CancellableFuture**|Template class used for retrieving the response to a synchronous call. This class has built in support for cancelling the synchronous call.|
+|**CancellationToken**|Control object to cancel asynchronous requests.|
+|**CancellableFuture**|Use Template class to retrieve the response to the synchronous call. This class has built-in support to cancel the synchronous call.|
```
Comment (andescu) [inline CONTRIBUTING.md]:
```suggestion
|**CancellableFuture**|Use this template class to retrieve the response to the synchronous call. This class has built-in support to cancel the synchronous call.|
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -294,27 +294,30 @@ class DATASERVICE_READ_API CatalogClient final {
...
+A sample component CMake files looks as follows:
 
+```
 <br />**project**(*olp-cpp-sdk*-**component** VERSION 1.0.0 DESCRIPTION "Component descripton")
 <br />...
 <br />// install component
 <br />**install** (FILES ${COMPONENT_INC} DESTINATION ${INCLUDE_DIRECTORY}/olp/component)
 <br />**export_config()**
-<br />  if(**OLP_SDK_ENABLE_TESTING**)
+<br />  if(**EDGE_SDK_ENABLE_TESTING**)
```
Comment (andescu) [inline CONTRIBUTING.md]: We changed all cmake defines from `EDGE_SDK_` to `OLP_SDK_` not so long ago.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ksenteer) [reply CONTRIBUTING.md]: Noted, thanks

**Pull request merged**